client/dashboard/utils/site-plans.ts

```diff
@@ -31,19 +31,23 @@ export const isSitePlanBigSkyTrial = ( site: Site ) => {
 };
 
 export const isSitePlanPaid = ( site: Site ) => {
-	if ( ! site.plan ) {
-		return false;
-	}
-
 	return ! [ DotcomPlans.JETPACK_FREE, DotcomPlans.FREE_PLAN ].includes(
-		site.plan.product_slug as DotcomPlans
+		site.plan?.product_slug as DotcomPlans
 	);
 };
 
 export const isSitePlanHostingTrial = ( site: Site ) => {
-	if ( ! site.plan ) {
-		return false;
-	}
+	return site.plan?.product_slug === DotcomPlans.HOSTING_TRIAL_MONTHLY;
+};
+
+export const isSiteOnECommerceTrial = ( site: Site ) => {
+	return site.plan?.product_slug === DotcomPlans.ECOMMERCE_TRIAL_MONTHLY;
```

Comment:
Channeling my inner Riad, are these `isPlan***` utils even that useful? Seems like all these functions are ever going to do is check the `product_slug` for one very specific paln.

Reply:
Right now they don't seem very useful other than aesthetic abstraction, happy to remove them for now.